notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1096 @@
+{
```

Comment:
<p>is there a reason the BQ source is None?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/984/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Yes. The data source can either be CSV (demoed in the notebook) or BigQuery and they are specified using different inputs. We've had some customers try to pass a BigQuery path using the CSV input leading to failed training jobs, so we put both inputs here to better clarify the usage, but set the BigQuery input to None as it's not being used. </p>